src/D2L.CodeStyle.Analyzers/RpcDependencies/RpcDependencyAnalyzer.cs

```diff
@@ -18,8 +18,18 @@ internal sealed class RpcDependencyAnalyzer : DiagnosticAnalyzer {
 			description: "RPCs must take an IRpcContext, IRpcPostContext or IRpcPostContextBase as their first argument"
 		);
 
+		internal static readonly DiagnosticDescriptor SortRule = new DiagnosticDescriptor(
+			id: "D2L0005",
+			title: "Dependency-injected arguments must be properly sorted",

```

Comment:
Dependency-injected arguments _in RPC methods_ must be properly sorted...
(just 'cause the title should be enough to disambiguate the error among diagnostics)